src/auth/guards/microsoft-auth.guard.ts

```
@@ -0,0 +1,9 @@
+import { Injectable } from '@nestjs/common';
+import { AuthGuard } from '@nestjs/passport';
+
+@Injectable()
+export class MicrosoftAuthGuard extends AuthGuard('microsoft') {
+  constructor() {
+    super();
+  }
```

Biome (1.9.4)

[error] This constructor is unnecessary.

Unsafe fix: Remove the unnecessary constructor.


(lint/complexity/noUselessConstructor)